kernel: bump 5.15 to 5.15.106

graysky2 opened the pull request:
Removed upstreamed:
	generic/735-net-ethernet-mtk_eth_soc-fix-flow_offload-related-re.patch[1]
	pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch[2]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.15.106&id=76f09582a191dcf11118fd4bdbf50f538c90fa8d

2. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/arch/mips/bmips?h=v5.15.106&id=65b723644294f1d79770704162c0e8d1f700b6f1

Build system: x86_64
Build-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod, ipq806x/R7800
Run-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod, ipq806x/R7800

CharlesMengCA reviewed (commented):
Tested working on mediatek/mt7622/linksys_e8450/ubi-squashfs-sysupgrade

DragonBluep commented on target/linux/generic/pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch:
```diff
@@ -135,39 +135,39 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
+ 	if (unlikely(bmips_rac_flush_disable))
 +		return;
 +
++	if (unlikely(bmips_rac_flush_disable))
```
This patch can be removed.
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/arch/mips/bmips?h=v5.15.106&id=65b723644294f1d79770704162c0e8d1f700b6f1

graysky2 replied:
You are correct, thanks.

hauke commented on target/linux/bmips/patches-5.15/200-mips-bmips-automatically-detect-CPU-frequency.patch:
```diff
@@ -58,17 +58,19 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
 +	const char		*compatible;
 +	u32			(*cpu_freq)(void);
-+};
++}
```
This change looks wrong to me. 
Why did your remove the `;`?

graysky2 replied:
I see now that removing the other patch mentioned by @dragonbluep and refreshing did not need modifications to this patch at all.

graysky2 committed:
kernel: bump 5.15 to 5.15.106

Removed upstreamed:
        generic/735-net-ethernet-mtk_eth_soc-fix-flow_offload-related-re.patch[1]
        pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch[2]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.15.106&id=76f09582a191dcf11118fd4bdbf50f538c90fa8d

2. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/arch/mips/bmips?h=v5.15.106&id=65b723644294f1d79770704162c0e8d1f700b6f1

Build system: x86_64
Build-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod, ipq806x/R7800
Run-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod, ipq806x/R7800

Signed-off-by: John Audia <therealgraysky@proton.me>
